packages/replay/src/canvas.ts

```diff
@@ -19,16 +28,20 @@ export class ReplayCanvas implements Integration {
    */
   public name: string;
 
+  public canvasManager: (options: GetCanvasManagerOptions) => CanvasManagerInterface;
+
   private _canvasOptions: ReplayCanvasOptions;
 
   public constructor() {
     this.name = ReplayCanvas.id;
     // TODO FN: Allow to configure this
     // But since we haven't finalized how to configure this, this is predefined for now
     // to avoid breaking changes
+    this.canvasManager = getCanvasManager;
     this._canvasOptions = {
-      fps: 4,
-      quality: 0.6,
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+      canvasQuality: CANVAS_QUALITY.medium,
```

Comment:
I would rename this in case we have other options in the future:

```suggestion
      canvasManagerOptions: CANVAS_QUALITY.medium,
```